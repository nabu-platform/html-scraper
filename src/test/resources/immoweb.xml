<instance id="main">
	<step action="cleanup"/>
	<step action="select" target="//td[@id='column-central']"/>
	<step action="before" target="/td/table" value="//ul[@class='description']"/>
	<step action="trim"/>
	<step action="drop" target="/td/table|//@target|//a/text()|//@id|//li[not(node())]|//li[@class='barre']" />
	<step action="rename" target="/td" value="result"/>
	<step action="rename" target="//ul" value="house"/>
	<step action="rename" target="//house/li[4]" value="city"/>
	<step action="rename" target="//house/li[3]" value="description"/>
	<step action="strip" target="//house/li[2]" />
	<step action="rename" target="//house/li[1]" value="price"/>
	<step action="extract" target="//a/@title"/>
	<step action="extract" target="//a/@href"/>
	<step action="rename" target="//href" value="link"/>
	<step action="copy" target="//link" value="id"/>
	<step action="substring" target="//id" value="[0-9]+"/>
	<step action="drop" target="//a|//@class|//li"/>
</instance>
