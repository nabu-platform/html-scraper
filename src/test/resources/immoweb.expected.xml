<result>
<house>
<price>99.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4490596&amp;xgallery=gallery&amp;xpage=1</link>
<id>4490596</id>
<title>Appartementsgeb. te koop met 2 slaapkamers te Antwerpen  - (4490596)</title>
<description> 1000 m²  - 2 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>101.500 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4595329&amp;xgallery=gallery&amp;xpage=1</link>
<id>4595329</id>
<title>Uitzonderlijk vastgoed te koop met 1 slaapkamer te Antwerpen  - (4595329)</title>
<description> 1000 m²  - 1 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>111.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4455392&amp;xgallery=gallery&amp;xpage=1</link>
<id>4455392</id>
<title>Appartementsgeb. te koop met 69 slaapkamers te Antwerpen  - (4455392)</title>
<description> 400 m²  - 69 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>149.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4490602&amp;xgallery=gallery&amp;xpage=1</link>
<id>4490602</id>
<title>Appartementsgeb. te koop met 2 slaapkamers te Antwerpen  - (4490602)</title>
<description> 1000 m²  - 2 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>165.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4224612&amp;xgallery=gallery&amp;xpage=1</link>
<id>4224612</id>
<title>Huis te koop met 2 slaapkamers te Antwerpen  - (4224612)</title>
<description> 120 m²  - 2 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>179.500 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4143648&amp;xgallery=gallery&amp;xpage=1</link>
<id>4143648</id>
<title>Huis te koop met 4 slaapkamers te Antwerpen 1  - (4143648)</title>
<description> 53 m²  - 4 slpk.</description>
<city>2000 Antwerpen 1</city>
</house>
<house>
<price>193.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4537146&amp;xgallery=gallery&amp;xpage=1</link>
<id>4537146</id>
<title>Gebouw gemengd gebruik te koop met 2 slaapkamers te Antwerpen  - (4537146)</title>
<description> 84 m²  - 2 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>198.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4133363&amp;xgallery=gallery&amp;xpage=1</link>
<id>4133363</id>
<title>Gebouw gemengd gebruik te koop met 2 slaapkamers te Antwerpen  - (4133363)</title>
<description> 132 m²  - 2 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>198.900 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4563754&amp;xgallery=gallery&amp;xpage=1</link>
<id>4563754</id>
<title>Gebouw gemengd gebruik te koop met 2 slaapkamers te Antwerpen  - (4563754)</title>
<description> 65 m²  - 2 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>199.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4490610&amp;xgallery=gallery&amp;xpage=1</link>
<id>4490610</id>
<title>Appartementsgeb. te koop met 2 slaapkamers te Antwerpen  - (4490610)</title>
<description> 1000 m²  - 2 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>199.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4595141&amp;xgallery=gallery&amp;xpage=1</link>
<id>4595141</id>
<title>Ander pand te koop met 1 slaapkamer te Antwerpen  - (4595141)</title>
<description> 217 m²  - 1 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>199.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4595140&amp;xgallery=gallery&amp;xpage=1</link>
<id>4595140</id>
<title>Huis te koop met 1 slaapkamer te Antwerpen  - (4595140)</title>
<description> 217 m²  - 1 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>199.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=3836067&amp;xgallery=gallery&amp;xpage=1</link>
<id>3836067</id>
<title>Huis te koop met 2 slaapkamers te Antwerpen  - (3836067)</title>
<description> 120 m²  - 2 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>200.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4557567&amp;xgallery=gallery&amp;xpage=1</link>
<id>4557567</id>
<title>Gebouw gemengd gebruik te koop te Antwerpen  - (4557567)</title>
<description> 51 m²</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>209.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4579931&amp;xgallery=gallery&amp;xpage=1</link>
<id>4579931</id>
<title>Huis te koop met 3 slaapkamers te Antwerpen  - (4579931)</title>
<description> 165 m²  - 3 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>232.200-295.605 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=3543043&amp;xgallery=gallery&amp;xpage=1&amp;xidbienbgi=3543048</link>
<id>3543043</id>
<title>Huis te koop met 2 slaapkamers te Antwerpen  - (3543048)</title>(4)<description>2 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>232.200 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4419330&amp;xgallery=gallery&amp;xpage=1</link>
<id>4419330</id>
<title>Huis te koop met 2 slaapkamers te Antwerpen  - (4419330)</title>
<description> 113 m²  - 2 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>237.600 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4419333&amp;xgallery=gallery&amp;xpage=1</link>
<id>4419333</id>
<title>Huis te koop met 3 slaapkamers te Antwerpen  - (4419333)</title>
<description> 116 m²  - 3 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>245.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4556630&amp;xgallery=gallery&amp;xpage=1</link>
<id>4556630</id>
<title>Huis te koop met 1 slaapkamer te Antwerpen  - (4556630)</title>
<description> 84 m²  - 1 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>245.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4595552&amp;xgallery=gallery&amp;xpage=1</link>
<id>4595552</id>
<title>Huis te koop met 1 slaapkamer te Antwerpen  - (4595552)</title>
<description> 90 m²  - 1 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>249.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4550393&amp;xgallery=gallery&amp;xpage=1</link>
<id>4550393</id>
<title>Gebouw gemengd gebruik te koop met 5 slaapkamers te Antwerpen  - (4550393)</title>
<description> 51 m²  - 5 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>250.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4545159&amp;xgallery=gallery&amp;xpage=1</link>
<id>4545159</id>
<title>Uitzonderlijk vastgoed te koop met 2 slaapkamers te Antwerpen  - (4545159)</title>
<description>2 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>250.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4427264&amp;xgallery=gallery&amp;xpage=1</link>
<id>4427264</id>
<title>Gebouw gemengd gebruik te koop te Antwerpen  - (4427264)</title>
<description> 190 m²</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>265.000-409.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4149354&amp;xgallery=gallery&amp;xpage=1&amp;xidbienbgi=4185330</link>
<id>4149354</id>
<title>Huis te koop met 3 slaapkamers te Antwerpen  - (4185330)</title>(7)<description>3 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>265.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4580067&amp;xgallery=gallery&amp;xpage=1</link>
<id>4580067</id>
<title>Huis te koop met 3 slaapkamers te Antwerpen  - (4580067)</title>
<description> 150 m²  - 3 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>265.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4559399&amp;xgallery=gallery&amp;xpage=1</link>
<id>4559399</id>
<title>Huis te koop met 2 slaapkamers te Antwerpen  - (4559399)</title>
<description> 105 m²  - 2 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>279.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4548772&amp;xgallery=gallery&amp;xpage=1</link>
<id>4548772</id>
<title>Uitzonderlijk vastgoed te koop met 4 slaapkamers te Antwerpen  - (4548772)</title>
<description> 240 m²  - 4 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>295.605 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4419334&amp;xgallery=gallery&amp;xpage=1</link>
<id>4419334</id>
<title>Huis te koop met 3 slaapkamers te Antwerpen  - (4419334)</title>
<description> 150 m²  - 3 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>296.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4583047&amp;xgallery=gallery&amp;xpage=1</link>
<id>4583047</id>
<title>Uitzonderlijk vastgoed te koop met 3 slaapkamers te Antwerpen  - (4583047)</title>
<description> 135 m²  - 3 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>299.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4516312&amp;xgallery=gallery&amp;xpage=1</link>
<id>4516312</id>
<title>Huis te koop met 2 slaapkamers te Antwerpen  - (4516312)</title>
<description> 152 m²  - 2 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>299.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4501086&amp;xgallery=gallery&amp;xpage=1</link>
<id>4501086</id>
<title>Herenhuis te koop met 4 slaapkamers te Antwerpen  - (4501086)</title>
<description> 250 m²  - 4 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>299.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4564963&amp;xgallery=gallery&amp;xpage=1</link>
<id>4564963</id>
<title>Huis te koop met 4 slaapkamers te Antwerpen  - (4564963)</title>
<description> 237 m²  - 4 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>299.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4162394&amp;xgallery=gallery&amp;xpage=1</link>
<id>4162394</id>
<title>Appartementsgeb. te koop te Antwerpen  - (4162394)</title>
<description> 150 m²</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>299.500 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4367500&amp;xgallery=gallery&amp;xpage=1</link>
<id>4367500</id>
<title>Gebouw gemengd gebruik te koop met 5 slaapkamers te Antwerpen  - (4367500)</title>
<description> 150 m²  - 5 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>305.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4563076&amp;xgallery=gallery&amp;xpage=1</link>
<id>4563076</id>
<title>Huis te koop met 3 slaapkamers te Antwerpen  - (4563076)</title>
<description> 180 m²  - 3 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>315.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4479122&amp;xgallery=gallery&amp;xpage=1</link>
<id>4479122</id>
<title>Huis te koop met 4 slaapkamers te Antwerpen  - (4479122)</title>
<description>4 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>325.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4577254&amp;xgallery=gallery&amp;xpage=1</link>
<id>4577254</id>
<title>Gebouw gemengd gebruik te koop te Antwerpen  - (4577254)</title>
<description> 148 m²</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>327.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4478702&amp;xgallery=gallery&amp;xpage=1</link>
<id>4478702</id>
<title>Huis te koop met 3 slaapkamers te Antwerpen  - (4478702)</title>
<description> 220 m²  - 3 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>329.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4479920&amp;xgallery=gallery&amp;xpage=1</link>
<id>4479920</id>
<title>Gebouw gemengd gebruik te koop met 2 slaapkamers te Antwerpen  - (4479920)</title>
<description> 180 m²  - 2 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>335.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4242002&amp;xgallery=gallery&amp;xpage=1</link>
<id>4242002</id>
<title>Gebouw gemengd gebruik te koop met 6 slaapkamers te Antwerpen  - (4242002)</title>
<description> 66 m²  - 6 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>335.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4242001&amp;xgallery=gallery&amp;xpage=1</link>
<id>4242001</id>
<title>Appartementsgeb. te koop met 6 slaapkamers te Antwerpen  - (4242001)</title>
<description> 66 m²  - 6 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>345.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4005354&amp;xgallery=gallery&amp;xpage=1</link>
<id>4005354</id>
<title>Huis te koop met 4 slaapkamers te Antwerpen  - (4005354)</title>
<description> 165 m²  - 4 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>345.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=3626733&amp;xgallery=gallery&amp;xpage=1</link>
<id>3626733</id>
<title>Huis te koop met 4 slaapkamers te Antwerpen  - (3626733)</title>
<description> 220 m²  - 4 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>346.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4487342&amp;xgallery=gallery&amp;xpage=1</link>
<id>4487342</id>
<title>Huis te koop met 4 slaapkamers te Antwerpen  - (4487342)</title>
<description> 150 m²  - 4 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>348.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4523932&amp;xgallery=gallery&amp;xpage=1</link>
<id>4523932</id>
<title>Huis te koop met 4 slaapkamers te Antwerpen  - (4523932)</title>
<description> 190 m²  - 4 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>348.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=4523931&amp;xgallery=gallery&amp;xpage=1</link>
<id>4523931</id>
<title>Huis te koop met 3 slaapkamers te Antwerpen  - (4523931)</title>
<description> 55 m²  - 3 slpk.</description>
<city>2000 Antwerpen</city>
</house>
<house>
<price>348.000 €</price>
<link>http://www.immoweb.be/nl/Buy.Estate.cfm?IdBien=3630856&amp;xgallery=gallery&amp;xpage=1</link>
<id>3630856</id>
<title>Herenhuis te koop met 4 slaapkamers te Antwerpen  - (3630856)</title>
<description> 240 m²  - 4 slpk.</description>
<city>2000 Antwerpen</city>
</house>
</result>
